balancer.go

```diff
@@ -83,11 +136,99 @@ func (info topicInfo) RoundRobin() map[string][]int32 {
 	return res
 }
 
+func rebalanceWithStrategy(topics map[string]TopicInfo,
+	rebalanceFunc func(TopicInfo) map[string][]int32,
+) map[string]map[string][]int32 {
+
+	res := make(map[string]map[string][]int32, 1)
+	for topic, info := range topics {
+		for memberID, partitions := range rebalanceFunc(info) {
+			if _, ok := res[memberID]; !ok {
+				res[memberID] = make(map[string][]int32, 1)
+			}
+			res[memberID][topic] = partitions
+		}
+	}
+	return res
+}
+
+// --------------------------------------------------------------------
+
+// StripedRebalancer implements StrategyStriped, balancing topics fairly
+// across consumers
+type StripedBalancer struct{}
```

Comment:
I was leaving these exposed as one could use them as a fallback in their own balancer implementation (i.e. use MyCustomBalancer under certain conditions, or fall back to StripedBalancer).